### PR TITLE
add modal popup for restart of job

### PR DESCRIPTION
**Pull request by lusol**
When user restarts a job, modal pops up to alert user their job is restarting.

screen shot 2018-05-17 at 3 37 12 pm

related pr: https://github.com/screwdriver-cd/screwdriver/issues/971

### CODE REVIEW

File: app/pipeline/build/controller.js [hunk 1]
```diff
@@ -40,9 +42,11 @@ export default Controller.extend({
 
       return newEvent.save().then(() =>
         newEvent.get('builds')
-          .then(builds =>
+          .then((builds) => {
             this.transitionToRoute('pipeline.build',
```
Comment (pranavrc) [inline app/pipeline/build/controller.js]: Earlier this was returning `this.transitionToRoute`, so move the `this.set('isShowingModal', false')` before this line and return the `this.transitionToRoute`

**Comment by pranavrc** [comment]
Might wanna add tests to `tests/unit/pipeline/build/controller-test.js`. Check `tests/unit/pipeline/index/controller-test.js` for how we test the same functionality.

File: app/pipeline/build/template.hbs [hunk 2]
```diff
@@ -27,4 +36,6 @@
 }}
 {{/if}}
 
+
+
```
Comment (pranavrc) [inline app/pipeline/build/template.hbs]: Unnecessary newlines

**Comment by tkyi** [comment]
Do you need this? https://github.com/screwdriver-cd/ui/pull/295/files#diff-fa51c1572a5862641cd834c260105899R56

**Pull request merged**